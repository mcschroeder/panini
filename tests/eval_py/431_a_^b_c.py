def f431(s: str):
  assert len(s) == 1
  c = s[0]
  if c != "a":
    if c != "c":
      assert c != "b"
