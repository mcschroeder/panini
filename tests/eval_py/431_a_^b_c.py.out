f431 : {s:𝕊 | s ∈ [^b]} → 𝟙